lambdas/journeyengine/src/main/java/uk/gov/di/ipv/core/journeyengine/JourneyEngineHandler.java

```diff
@@ -48,7 +53,8 @@
     private static final String KBV_CRI_ID = "kbv";
     private static final String FRAUD_CRI_ID = "fraud";
 
-    private static final List<String> VALID_JOURNEY_STEPS = List.of("next");
+    private static final List<String> VALID_JOURNEY_STEPS =
+            Stream.of(JourneyStep.values()).map(JourneyStep::toString).collect(Collectors.toList());
```

Comment:
I like the use of the enum's, I'm just wondering if rather than creating this list, we can use the enum itself. Something like this maybe? https://www.baeldung.com/java-search-enum-values#searching-an-enum-by-value
Section 5 on that page deals with throwing errors if the value isn't found which might also be useful.

Reply:
Might be easier to add this over on Dan's PR https://github.com/alphagov/di-ipv-core-back/pull/287 since he's already converted to an enum with his changes

Reply:
Might be easier to add this over on Dan's PR https://github.com/alphagov/di-ipv-core-back/pull/287 since he's already converted to an enum with his changes

Reply:
I like this idea - will implement!